lib/peep.rb

```diff
@@ -0,0 +1,36 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
Could put this if/else statement into a method of its own as used multiple times